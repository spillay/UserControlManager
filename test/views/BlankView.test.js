import React from 'react';
import BlankView from '../../src/views/BlankView.jsx'
import renderer from 'react-test-renderer';

test('BlankView test', () => {
  const component = renderer.create(
    <BlankView modelForm="test"/>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

//   // manually trigger the callback
//   tree.props.onMouseEnter();
//   // re-rendering
//   tree = component.toJSON();
//   expect(tree).toMatchSnapshot();

//   // manually trigger the callback
//   tree.props.onMouseLeave();
//   // re-rendering
//   tree = component.toJSON();
//   expect(tree).toMatchSnapshot();
});