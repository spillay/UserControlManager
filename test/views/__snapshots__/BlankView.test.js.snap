// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlankView test 1`] = `
<div>
  Blank View for Testing Purposes
</div>
`;
